website_multi_image/README.rst

```diff
@@ -0,0 +1,10 @@
+Product Multi-Image
+===================
+
+This module adds multiple product images into
+a tab in product.template in a tab called
+Product Images, and allows an over-ride
+of product.image to allow for a multi-image
+carousel for the product view page.
+
+web_tree_image is recommended to install to get images preview: https://github.com/OCA/web/tree/8.0/web_tree_image
```

Comment:
I am agreed to install this another module in vx/yoy it is necesary in order to see the image in the list vie in the product form. Can you add such dependency in the dummy PR also please.